askama_shared/src/parser.rs

```diff
@@ -863,6 +871,21 @@ fn block_let(i: &[u8]) -> IResult<&[u8], Node<'_>> {
     ))
 }
 
+fn else_block<'a>(i: &'a [u8], s: &'a Syntax<'a>) -> IResult<&'a [u8], (Ws, Vec<Node<'a>>)> {
```

Comment:
What is the rationale for extracting this from `block_for`? If we're going to extract it, can we share it with `if`?

Reply:
There's no good reason other than readability. I'll turn it into a closure.